docs/proposals/proposal-trigger-and-deploy.md

```diff
@@ -73,14 +73,58 @@ we will then deploy the benchmarking workfload for the project, in this case its
 
 ## Proposal
 
-<!--
-This is where we get down to the specifics of what the proposal actually is:
-outlining your solution to the problem described in the Motivation section.
-This should have enough detail that reviewers can understand exactly what
-you're proposing, but should not include things like API designs or
-implementation. The "Design Details" section below is for the real
-nitty-gritty.
--->
+got an example GitHub action workflow file
+```yaml
+name: TriggerTest
+
+on:
+  workflow_dispatch:
+    inputs:
+      cncf_project:
+        description: 'CNCF Project Name'
+        required: true
+        default: 'falco'
+      cncf_project_sub:
+        description: 'CNCF Project Subcomponent'
+        required: false
+        default: 'modern-ebpf'
+      version:
+        description: 'Version'
+        required: true
+        default: '0.37.0'
+
+jobs:
+  echo-inputs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Echo Inputs
+      run: |
+        echo "Add logic to deploy ${{ github.event.inputs.cncf_project }} ${{ github.event.inputs.cncf_project_sub }}"
+        echo "version ${{ github.event.inputs.version }}"
+```
+
+for invoking this
+
+```bash
+curl -X POST \
+     -H "Accept: application/vnd.github.v3+json" \
+     -H "Authorization: token $GITHUB_PAT" \
+     https://api.github.com/repos/rossf7/green-reviews-tooling/actions/workflows/trigger_test.yaml/dispatches \
+     -d '{"ref":"main", "inputs": {"cncf_project": "falco", "cncf_project_sub": "modern-ebpf","version":"0.37.0"}}'
```

Comment:
For the ref maybe we should use a tag here?

The docs on release management are pretty good

https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions

Although I think some recommendations like having the action in a separate repo are too heavy for our needs

Reply:
> Although I think some recommendations like having the action in a separate repo are too heavy for our needs

Yes 

> For the ref maybe we should use a tag here?

yes